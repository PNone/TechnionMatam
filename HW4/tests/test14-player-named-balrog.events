Slime
Pack 2 Slime Pack 2 Pack 2 Snail Snail Slime
SolarEclipse
PotionsMerchant