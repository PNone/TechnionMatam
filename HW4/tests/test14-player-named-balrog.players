Balrog Warrior Responsible
Baraa Archer RiskTaking