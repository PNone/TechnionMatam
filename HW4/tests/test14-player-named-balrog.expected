Game is about to start!
These are the players:

1. Balrog, Warrior with Responsible character (level 1, force 5)
2. Baraa, Archer with RiskTaking character (level 1, force 5)

----------------------------------------

New round starts!

Turn: 1
Player: Balrog, Warrior with Responsible character (level 1, force 5)
Event: Slime (power 12, loot 5, damage 25)
Balrog lost the encounter and took 25 damage!

Turn: 2
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: Pack of 2 members (power 34, loot 14, damage 70)
Baraa lost the encounter and took 70 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     30    20
2          Balrog          1         5     115   10

----------------------------------------

New round starts!

Turn: 3
Player: Balrog, Warrior with Responsible character (level 1, force 5)
Event: SolarEclipse
Balrog was affected by a solar eclipse! their force has changed by -1!

Turn: 4
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 1 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     40    15
2          Balrog          1         4     115   10

----------------------------------------

New round starts!

Turn: 5
Player: Balrog, Warrior with Responsible character (level 1, force 4)
Event: Slime (power 12, loot 5, damage 25)
Balrog lost the encounter and took 25 damage!

Turn: 6
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: Pack of 2 members (power 34, loot 14, damage 70)
Baraa lost the encounter and took 70 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     15
2          Balrog          1         4     80    10

----------------------------------------

New round starts!

Turn: 7
Player: Balrog, Warrior with Responsible character (level 1, force 4)
Event: SolarEclipse
Balrog was affected by a solar eclipse! their force has changed by -1!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     15
2          Balrog          1         3     80    10

----------------------------------------

New round starts!

Turn: 8
Player: Balrog, Warrior with Responsible character (level 1, force 3)
Event: PotionsMerchant
Balrog bought 2 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     15
2          Balrog          1         3     100   0

----------------------------------------

New round starts!

Turn: 9
Player: Balrog, Warrior with Responsible character (level 1, force 3)
Event: Slime (power 12, loot 5, damage 25)
Balrog lost the encounter and took 25 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     15
2          Balrog          1         3     65    0

----------------------------------------

New round starts!

Turn: 10
Player: Balrog, Warrior with Responsible character (level 1, force 3)
Event: Pack of 2 members (power 34, loot 14, damage 70)
Balrog lost the encounter and took 70 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     15
2          Balrog          1         3     0     0

----------------------------------------

Game is over!
There are no winners in this game.
Better luck next time!
